components/linkedin/linkedin.app.mjs

```diff
@@ -1,11 +1,158 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
 export default {
   type: "app",
   app: "linkedin",
-  propDefinitions: {},
+  propDefinitions: {
+    text: {
+      type: "string",
+      label: "Text",
+      description: "Text to be posted on LinkedIn timeline",
+    },
+    organizationId: {
```

Comment:
I think `organizationId` prop can have `async options`, so that users can select from organizations. [This](https://github.com/PipedreamHQ/pipedream/blob/f6ce80eac4520ab147fba65eb8ab08054b2e987d/components/microsoft_outlook/microsoft_outlook.app.mjs#L39) can be an example.
Applying pagination might also be good here.